build.ps1

```diff
@@ -1,3 +1,4 @@
 #!/usr/bin/env pwsh
 
-go build -v -o pct.exe
+# Set goreleaser to build for current platform only
+goreleaser build --snapshot --rm-dist --single-target
```

Comment:
I've been having a think about this. 

Should we make `build.(ps1|sh)` always use the `--skip-publish` option to build the binaries and then the `release.yml` GHActions workflow will be our only method to publish a new artefact?

Reply:
'goreleaser build' never publishes